problem_builder/public/js/lms_util.js

```diff
@@ -0,0 +1,109 @@
+(function () {
```

Comment:
@itsjeyd  `utils.js` is used only in studio (or `cms`), and this one is used only in `lms` and while this would have negligible effect on performance, i decided that adding unneeded code would be bad, so I decided to split them. 

I'll rename them `step_utils.js`.